src/bot/root_pythia_cogs.py

```diff
@@ -146,6 +147,16 @@ async def verbose_if_idle(self, channel):
 
     @commands.Cog.listener()
     async def on_command_error(self, ctx, error):
+        if isinstance(error, CheckFailure):
```

Comment:
I am not sure to understand what this part does. Could you explain it in a few words ?

Reply:
`on_command_error` is a "callback" it's a function automatically called whenever the bot encounters an exception when responding to a command triggered by users.
The parameter error is the exception raised
So basically I implemented this function to send a message in the dfiscord channel when a command fails

Here I check if this exception is a `CheckFailure` type if so I don't send a message to the disocrd channel like the default behavior to avoid flooding becaue I assumes it has been raised by the global check I added (becaue it is the only check for now...)

Reply:
Oh okay I think I get it now.
Thanks !